src/edge-core-index.js

```diff
@@ -543,6 +543,7 @@ export type EdgeSpendTarget = {
   publicAddress?: string,
   nativeAmount?: string,
   destMetadata?: EdgeMetadata,
+  sAddress?: string,
```

Comment:
This shouldn't be needed here. Only `otherParams.uniqueIdentifier`

Reply:
actually I need it to pass flow, because it comes back from shapeshift and is part of the object that is returned, so when I assign uniqueIdentifier to returnedData.sAddress if it is not defined here it does not pass flow